plugins/hockey-scoreboard/data_fetcher.py

```
@@ -54,10 +54,10 @@ def __init__(self, cache_manager, logger: logging.Logger):
     
     def _get_season_date_range(self, league_key: str) -> str:
         """Get the date range for the current season."""
-        from datetime import datetime, timedelta
+        from datetime import datetime
         
         now = datetime.now()
-        current_year = now.year
+        now.year
```

Ruff (0.15.12)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)